example_payloads/qc_metrics.vc.json

```diff
@@ -0,0 +1,137 @@
+{
+  "studyId": "TEST-PR",
+  "analysisType": {
+    "name": "qc_metrics",
+    "version": 7
+  },
+  "samples": [
+    {
+      "submitterSampleId": "COLO-829",
+      "matchedNormalSubmitterSampleId": "COLO-829-BL",
+      "sampleType": "Total DNA",
+      "specimen": {
+        "submitterSpecimenId": "COLO-829",
+        "tumourNormalDesignation": "Tumour",
+        "specimenTissueSource": "Blood derived",
+        "specimenType": "Cell line - derived from tumour"
+      },
+      "donor": {
+        "gender": "Female",
+        "submitterDonorId": "COLO-829"
+      }
+    }
+  ],
+  "files": [
+    {
+      "info": {
```

Comment:
this `info` field seems not the latest, something like below? [same comment as in other info fields]
```
"info": {
    "analysis_tools": [
        "verifyBamHomChk"
    ],
    "data_category": "Quality Control Metrics",
    "description": "Cross sample contamination estimated by Sanger verifyBamHomChk.pl script",
    "files_in_tgz": [
        "SA528843_snps.vcf",
        "SA528843.log",
        "SA528843.selfSM",
        "SA528843.selfRG",
        "SA528843.depthSM",
        "SA528843.depthRG",
        "result.json"
    ],
    "metrics": {
        "avg_depth": 57.92,
        "contamination": 0.00004,
        "reads_used": 2043286,
        "sample_id": "SA528843",
        "snps_used": 35275
    }
}
```